app/controllers/profiles_controller.rb

```diff
@@ -38,11 +42,6 @@ def changed_email?
     user_params[:email].present?
   end
 
-  def set_mail_changes
-    flash[:notice] = I18n.t('flash.notice.profile_email_updated')
-    force_send_confirmation_email if @user.email.blank?
```

Comment:
👍  There's an `email` presence validation in `User`, so this would never be triggered.